iron-fit-behavior.html

```diff
@@ -485,14 +485,19 @@
     },
 
     __getNormalizedRect: function(target) {
-      if (target === document.documentElement || target === window) {
+      if (target === window) {
```

Comment:
Following that, I don't think any of the changes below are necessary. `window` just indicates the viewport here and `getBoundingClientRect` (which this method seeks to extend to `window`) is always relative to the viewport. But, the viewport's coordinates relative to itself to its own top and left are unchanged. `st` and `sl` should always be zero.